modules/calendar/add_edit_event.php

```diff
@@ -1912,7 +2048,7 @@ function SubmitForm() {
 
 $('#reasons_modal').iziModal({
                title: "<?php echo xlt('Reason for Cancellation'); ?>",
-               subtitle: "<?php echo xlt('Choose the reason for Cancellation. If the reason is not listed then please use the other'); ?>",
+               subtitle: "<?php echo xlt('Choose the reason for Cancellation.if the reason is not listed then please use other'); ?>",
```

Comment:
'Choose the reason for Cancellation.  If none are applicable, chose [Other]'

Reply:
Additional typo introduced when fixing the language.

Reply:
Change made

Reply:
oops!  *choose* [Other]
I typodeded two.

Reply:
Not seeing the change.  PUUUUSH